aptos-move/aptos-vm/src/natives.rs

```diff
@@ -11,5 +12,6 @@ pub fn aptos_natives() -> NativeFunctionTable {
         .into_iter()
         .chain(framework::natives::all_natives(CORE_CODE_ADDRESS))
         .chain(move_table_extension::table_natives(CORE_CODE_ADDRESS))
+        .chain(transaction_context_natives(CORE_CODE_ADDRESS))
```

Comment:
I wonder if we should also expose the sender of the transaction... there have been some security ideas I've been toying with.

the other thing I was pondering is whether or not we have clear concept of what belongs in the framework versus here.

Reply:
Yes there's a lot of potential for this transaction context (that's why I named it so). We can expose sender, gas limit, etc. that would be really useful for modules to understand the transaction context.